src/middleware/queriesV2.ts

```diff
@@ -88,15 +88,15 @@ function createClaimsFilters(variables: ClaimsQueryFilter): {
   const args = ['$size: Int, $start: Int'];
   const filters = ['revokeDate: { isNull: true }'];
   const { dids, claimTypes, trustedClaimIssuers, scope, includeExpired } = variables;
-  if (dids) {
+  if (dids && dids.length) {
```

Comment:
This seems better than filtering for empty array which is trivially the empty set.